examples/google-app-engine/README.md

```diff
@@ -0,0 +1,72 @@
+Example project for Google App Engine
+=====================================
+
+This directory contains example projects demonstrating the use of gRPC in Google App Engine (GAE).
+At this time, only gRPC clients are supported in GAE. gRPC servers are not supported in GAE.
+
+Java 7 vs Java 8
+================
+
+The GAE Java 8 runtime has fewer restrictions than the Java 7 runtime, and supports the standard
+public Java library. This allows the netty-transport to be used and allows servlets to have
+long lived transport objects, rather than creating a new one for each servlet request.
+
+In Java 7, the okhttp-transport must be used and each servlet request must create a new transport.
+
+Prerequisite knowledge
+===================
+
+* [Quickstart for Java 8 for App Engine Standard Environment](https://cloud.google.com/appengine/docs/standard/java/quickstart-java8)
+* [Quickstart for Java 7 for App Engine Standard Environment] https://cloud.google.com/appengine/docs/standard/java/quickstart
+* [Using Gradle and the App Engine Plugin](https://cloud.google.com/appengine/docs/standard/java/tools/gradle)
+
+Running the example locally
+===========================
+
+```bash
+# From the root gRPC Java directory, install the current snapshot to local .m2
```

Comment:
Please no. Trying to compile gRPC is a horrible experience for new users. We don't include this step in our examples. Instead, we expect users to check out the version tag. We have this in examples/README.md:

> You are strongly encouraged to check out a git release tag